apps/extension/src/utils/index.ts

```diff
@@ -115,3 +122,36 @@ export const isCustomPath = (path: Path): boolean => {
   }
   return false;
 };
+
+/**
+ * Create label to indicate specific type of Transfer
+ * @param tx TransferProps
+ * @returns string label
+ */
+export const parseTransferType = (
+  tx: TransferProps
+): { source: string; target: string; type: TransferType } => {
+  const { sources, targets } = tx;
+  const source = sources[0].owner;
+  const target = targets[0].owner;
+
+  const shieldedPoolAddress = "tnam1pcqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqzmefah";
```

Comment:
should we pull this from namada-chain-registry?

Reply:
@euharrison I'll take a look! The extension doesn't touch the registry at all yet, but perhaps it's time it does, so long as it doesn't add too much bulk to the build size. The MASP address will indeed need to match on any given chain, so I think that's the only way to do it. Thanks!

Reply:
Updated this to not even need to use the shielded pool address!